tests/std/tests/P0898R3_concepts/test.cpp

```diff
@@ -1543,7 +1543,7 @@ namespace test_default_initializable {
         int x;
     };
     STATIC_ASSERT(default_initializable<S>);
-#if defined(MSVC_INTERNAL_TESTING) || defined(__clang__) || defined(__EDG__) // TRANSITION, DevCom-952724
+#if defined(_MSVC_INTERNAL_TESTING) || defined(__clang__) || defined(__EDG__) // TRANSITION, DevCom-952724

```

Comment:
DevCom-952724 is "Fixed In: Visual Studio 2022 version 17.0". Could this #if guard be removed? (This removal can be done in a different PR, of course.)

Reply:
Yes. We need to either remove the guard, or file a new bug report and update the annotation. I'd prefer not to do so in this PR since it's already been partially reviewed.

Reply:
If you're motivated, feel free to submit a PR and we'll fix the merge conflict if this one lands first.

Reply:
#2351